app/src/androidMain/res/values/strings.xml

```diff
@@ -723,6 +723,10 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_atm_cashin_title">"Can you deposit cash at this ATM?"</string>
     <string name="quest_atm_operator_title">"Which bank operates this ATM?"</string>
 
+    <string name="quest_atp_add_missing_poi_title">This place may be here - where exactly it is located? Tap on its exact location</string>
+    <string name="quest_atp_add_missing_poi_mapped_already">"This place is mapped already"</string>
+    <string name="quest_atp_add_missing_poi_does_not_exist">"This place does not exist"</string>
```

Comment:
Instructions on how to answer the quest should not be in the question but in the quest form body.

Also, IMO better: "It's already displayed on the map" and "No such place here". 🤷

Reply:
> Instructions on how to answer the quest should not be in the question but in the quest form body.

BTW, I was thinking about migrating to cross selector like in things overlay and places overlay (not sure have you tried using or looked at code for now)